src/modules/form.js

```diff
@@ -0,0 +1,73 @@
+// Add these lines at the top of the file
+import { todoLists, displayTodoList } from './todos.js';
+
+const form = document.getElementById('todoForm');
+const todoInput = document.getElementById('newtodo');
+const errorMessag = document.getElementById('errorMessag');
+const todosListParent = document.getElementById('todo-list');
+
+// Define the removeTodo function
+// eslint-disable-next-line no-use-before-define
+const removeTodo = (index) => { // eslint-disable-line
+  todoLists.splice(index, 1);
+  displayTodoList();
+  localStorage.setItem('todoLists', JSON.stringify(todoLists));
+};
+
+// add event listener to editBtn
+todosListParent.addEventListener('click', (e) => {
```

Comment:
- [ ] The edit is not working properly, when I click on the edit button it throws this error in the browser, kindly make sure to fix this so users can modify the tasks

![Screenshot_20230623_000423](https://github.com/akbarsazish/To-Do-List/assets/25227022/ff90f394-a6be-4c9a-b64c-554c7121d1a0)